cmd/kubectl_token.go

```diff
@@ -510,58 +532,84 @@ func getAuthProviders(server string) (map[string]string, error) {
 		return nil, err
 	}
 
-	data := map[string]interface{}{}
-	err = json.Unmarshal(response, &data)
-	if err != nil {
-		return nil, err
+	if !gjson.ValidBytes(response) {
+		return nil, errors.New("invalid JSON input")
```

Comment:
How about "response" here rather than "input"? and could we include the `authProviderURL`  which might point to why?